test/metabase/events/view_log_test.clj

```diff
@@ -62,21 +62,26 @@
              (latest-view (mt/user->id :crowberto) (u/id coll))))))))
 
 (deftest update-view-dashboard-timestamp-test
-  (let [now           (t/offset-date-time)
-        one-hour-ago  (t/minus now (t/hours 1))
-        two-hours-ago (t/minus now (t/hours 2))]
+  (let [now            (t/offset-date-time)
+        one-hour-ago   (t/minus now (t/hours 1))
+        two-hours-ago  (t/minus now (t/hours 2))
+        ;; `(t/offset-date-time)` has nanosecond precision on my JVM while app DB timestamp columns on my machine seem
+        ;; to have microsecond precision.
+        truncate-to-ms (fn [t]
+                         (.truncatedTo (t/offset-date-time t) java.time.temporal.ChronoUnit/MILLIS))]
     (testing "update with multiple dashboards of the same ids will set timestamp to the latest"
       (mt/with-temp
         [:model/Dashboard {dashboard-id-1 :id} {:last_viewed_at two-hours-ago}]
         (#'events.view-log/update-dashboard-last-viewed-at!* [{:id dashboard-id-1 :timestamp one-hour-ago}
                                                               {:id dashboard-id-1 :timestamp two-hours-ago}])
-        (is (= one-hour-ago (t2/select-one-fn :last_viewed_at :model/Dashboard dashboard-id-1)))))
-
+        (is (= (truncate-to-ms one-hour-ago)
```

Comment:
This test and one other test were failing for me locally. `(t/offset-date-time)` has nanosecond precision on my JVM while app DB timestamp columns on my machine seem to have microsecond precision. Unrelated, but I fixed these tests.